Assets/MixedRealityToolkit/Services/MixedRealityToolkit.cs

```diff
@@ -233,6 +233,11 @@ public void ResetConfiguration(MixedRealityToolkitConfigurationProfile profile)
             if (registeredMixedRealityServices.Contains(registryInstance))
             {
                 registeredMixedRealityServices.Remove(registryInstance);
+                if (!(serviceInstance is IMixedRealityDataProvider))
+                {
+                    // Only remove IMixedRealityService or IMixedRealityExtensionService (not IMixedRealityDataProvider)
```

Comment:
>// Only remove IMixedRealityService or IMixedRealityExtensionService (not IMixedRealityDataProvider) [](start = 20, length = 100)

Nit: Can you update this comment to explain the 'why' part (instead of the 'what') - it is clear from the code that it's not doing things to data providers (this is what the comment also says) but what's actually valuable is to leave a note about why this is being done.

Reply:
Sounds good. I will clarify the comment when i move the check to AddService